src/__test__/table-theme-colors.spec.js

```diff
@@ -0,0 +1,481 @@
+      it('should return the when the background color on object is dark and opaque', () => {
+        global.document.querySelector = jest.fn((selector) => selector);
+        global.window.getComputedStyle = jest.fn((selector) => {
+          let style;
+          if (selector === '.qv-object .qv-inner-object') {
+            style = { backgroundColor: '#000' };
+          } else if (selector === '.qv-panel-sheet') {
+            style = { backgroundColor: '#fff' };
+          }
+          return style;
+        });
+        const result = tableThemeColors(theme);
+        expect(result).toEqual({ ...darkBackgroundColorValue, backgroundColor: '#000' });
+      });
+
+      it('should return the when the background color on object is light and opaque', () => {
+        global.document.querySelector = jest.fn((selector) => selector);
+        global.window.getComputedStyle = jest.fn((selector) => {
+          let style;
+          if (selector === '.qv-object .qv-inner-object') {
+            style = { backgroundColor: '#fff' };
+          } else if (selector === '.qv-panel-sheet') {
+            style = { backgroundColor: '#000' };
+          }
+          return style;
+        });
+        const result = tableThemeColors(theme);
+        expect(result).toEqual({ ...defaultValue, backgroundColor: '#fff' });
+      });
+
+      it('should return the when the background color on object is transparent and background color on sheet is light', () => {
+        global.document.querySelector = jest.fn((selector) => selector);
+        global.window.getComputedStyle = jest.fn((selector) => {
+          let style;
+          if (selector === '.qv-object .qv-inner-object') {
+            style = { backgroundColor: 'rgba(0,0,0,0)' };
+          } else if (selector === '.qv-panel-sheet') {
+            style = { backgroundColor: '#fff' };
+          }
+          return style;
+        });
+        const result = tableThemeColors(theme);
+        expect(result).toEqual({
+          ...defaultValue,
+          backgroundColor: 'rgba(0,0,0,0)',
+          isBackgroundTransparentColor: true,
+        });
+      });
+
+      it('should return the when the background color on object is transparent and background color on sheet is dark', () => {
+        global.document.querySelector = jest.fn((selector) => selector);
+        global.window.getComputedStyle = jest.fn((selector) => {
+          let style;
+          if (selector === '.qv-object .qv-inner-object') {
+            style = { backgroundColor: 'rgba(0,0,0,0)' };
+          } else if (selector === '.qv-panel-sheet') {
+            style = { backgroundColor: '#000' };
+          }
+          return style;
+        });
+        const result = tableThemeColors(theme);
+        expect(result).toEqual({
+          ...darkBackgroundColorValue,
+          backgroundColor: 'rgba(0,0,0,0)',
+          isBackgroundTransparentColor: true,
+        });
+      });
+    });
+  });
```

Comment:
You need to check  `isBackgroundDarkColor` otherwise the test check the same thing

Reply:
Sorry, removed it and will be up in another PR.